### PR TITLE
[fix] exit if not runned as root instead of raising a obscur exception

### PR DESCRIPTION
**Pull request by Psycojoker**

### CODE REVIEW

**Comment by M5oul** [comment]
Nice PR!

**Comment by jeromelebleu** [comment]
Merged with https://github.com/YunoHost/yunohost/commit/8a6b31b69043f5aeca28a972ddc9a9ab6bf63f6f, thanks!

File: bin/yunohost [hunk 1]
```diff
@@ -185,6 +185,10 @@ def _retrieve_namespaces():
 # Main action ----------------------------------------------------------
 
 if __name__ == '__main__':
+    if os.geteuid() != 0:
+        # since moulinette isn't initialized, we can't use m18n here
+        sys.stderr.write("\033[1;31mError:\033[0m yunohost command must be runed as root or with sudo.\n")
```
Comment (julienmalik) [inline bin/yunohost]: must be *run* as root :)

Comment (jeromelebleu) [inline bin/yunohost]: Yes, fixed in the merged commit, see https://github.com/YunoHost/yunohost/commit/8a6b31b69043f5aeca28a972ddc9a9ab6bf63f6f#diff-3860ca91a6d504392d1fcf2390c4eab7R191 :)

**Pull request closed without merging**